OpenRA.Mods.Common/Traits/Air/AttackAircraft.cs

```diff
@@ -33,6 +33,9 @@ public class AttackAircraftInfo : AttackFollowInfo, Requires<AircraftInfo>
 		[Desc("Does this actor cancel its attack activity when it needs to resupply? Setting this to 'false' will make the actor resume attack after reloading.")]
 		public readonly bool AbortOnResupply = true;
 
+		[Desc("Facing to attack")]
```

Comment:
`The angle relative to the actor's orientation used to fire the weapon from.`? (That doesn't sound great, so I'm open to suggestions.)

Reply:
Fixed. the more detailed, the better